keyboards/kinesis/kint41/keymaps/jtojnar/keymap.c

```diff
@@ -0,0 +1,103 @@
+#include QMK_KEYBOARD_H
+
+enum my_layers {
+    // Base QWERTY
+    QWERTY = 0,
+    // Keypad layer
+    KEYPAD = 1,
```

Comment:
@stapelberg do you think it makes sense to incorporate some of these changes to the default keymap?

Reply:
* A keypad layer seems reasonable, and I have recently implemented one in my personal keymap: https://github.com/qmk/qmk_firmware/pull/18295
* The left super instead of right super change seems reasonable, too.
* Moving the bootloader button out of the way is fine with me, but not sure if some users are used to it.

The other commits seem more like personal preference to me.